src/Drivers/ExchangeRateHost/ExchangeRateHostDriver.php

```diff
@@ -19,7 +19,7 @@ class ExchangeRateHostDriver implements ExchangeRateDriver
 
     private SharedDriverLogicHandler $sharedDriverLogicHandler;
 
-    public function __construct(RequestBuilder $requestBuilder = null, CacheRepository $cacheRepository = null)
+    public function __construct(?RequestBuilder $requestBuilder = null, ?CacheRepository $cacheRepository = null)
```

Comment:
What's your opinion on changing these so they are not nullable at all and have to be passed in? Like this:

```php
public function __construct(RequestBuilder $requestBuilder, CacheRepository $cacheRepository)
```

It's something I added ages ago when I first created this package (when the drivers weren't hidden by a manager class), and I just don't feel like it provides any benefit anymore.

I don't know if I'm just causing more work myself (and potentially causing issues for devs using the package) though 🤔

Reply:
It would be a good moment to do it, but I'm not sure if its worth it